ui/app/components/ui/icon-button/icon-button.js

```diff
@@ -28,7 +28,7 @@ export default function IconButton ({ onClick, Icon, disabled, label, tooltipRen
 
 IconButton.propTypes = {
   onClick: PropTypes.func.isRequired,
-  Icon: PropTypes.element.isRequired,
+  Icon: PropTypes.oneOfType([PropTypes.element, PropTypes.func]).isRequired,
```

Comment:
This should always be a function - never an element. `PropTypes.element` is for _instances_ of React elements (e.g. `<Icon>`), not for the component class/function itself.
```suggestion
  Icon: PropTypes.func.isRequired,
``

Reply:
db1b2db